index.d.ts

```diff
@@ -181,6 +181,19 @@ export interface ActionConfig<TData extends object = object> {
    * @default true
    */
   abortOnFail?: boolean;
+	/**
+	 * Skip an action if this function returns a string,
+	 * which is the reason it should be skipped.
+	 *
+	 * May also return a Promise which resolves to a string.
+	 *
+	 * The action will continue if action.when()
+	 * returns or resolves to anything other than a string,
+	 * and the return value will be ignored.
+	 *
+	 * @default () => true
+	 */
+	when?: (data: TData) => void | string | Promise<void | string>;
```

Comment:
This is super super minor, but I'm noticing that the spacing here isn't consistent with the rest of the `.d.ts` file. Are we able to make that more consistent?

Reply:
Been thinking about formatting, should we add Prettier to a precommit hook?